Cargo.lock

```diff
@@ -1246,9 +1043,9 @@ dependencies = [
 
 [[package]]
 name = "vmm-sys-util"
-version = "0.11.1"
+version = "0.11.2"
```

Comment:
Why `vmm-sys-util` is upgrading, I don't think it is a dependency of `config` or `serde`.

Did you run a generic `cargo update`?

Reply:
Yeah, I had. I uploaded the another patchset with just cargo build.